### PR TITLE
L2 - Emotion

### PR DESCRIPTION
**Pull request by ryanmccombe**
Relates to #3763

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-headings/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.3 | [PR#3890](https://github.com/bbc/psammead/pull/3890) Migrate to emotion |
```
Comment (ryanmccombe) [inline packages/components/psammead-headings/CHANGELOG.md]:
```suggestion
| 5.0.0 | [PR#3890](https://github.com/bbc/psammead/pull/3890) Migrate to emotion |
```

File: packages/components/psammead-social-embed/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 # Social Embed Changelog
 
-| Version       | Description                                                                                               |
-| ------------- | --------------------------------------------------------------------------------------------------------- |
-| 2.0.3 | [PR#3883](https://github.com/bbc/psammead/pull/3883) Talos - Bump Dependencies - @bbc/psammead-styles |
-| 2.0.2 | [PR#3855](https://github.com/bbc/psammead/pull/3855) Talos - Bump Dependencies - @bbc/gel-foundations |
+| Version       | Description                                                                                                 |
+| ------------- | ----------------------------------------------------------------------------------------------------------- |
+| 3.0.0         | [PR#3890](https://github.com/bbc/psammead/pull/?) Migrate to Emotion                                        |
```
Comment (simonsinclair) [inline packages/components/psammead-social-embed/CHANGELOG.md]:
```suggestion
| 3.0.0         | [PR#3890](https://github.com/bbc/psammead/pull/3890) Migrate to Emotion                                        |
```

**Review by simonsinclair — APPROVED** [review]
Almost perfect.

File: packages/components/psammead-paragraph/package.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
     "@bbc/psammead-styles": "^6.0.0"
   },
   "peerDependencies": {
-    "styled-components": "^4.3.2"
+    "styled-components": "^4.3.2",
```
Comment (RichardPK) [inline packages/components/psammead-paragraph/package.json]: Do we still want this?

**Review by RichardPK — APPROVED** [review]
Just that peer dependency 👍

File: packages/components/psammead-paragraph/package.json (hunk 3 above)
Comment (ryanmccombe) [reply packages/components/psammead-paragraph/package.json]: Absolutely not 🦀

**Pull request merged**